src/declarativewidgets_plugin.cpp

```diff
@@ -68,6 +68,7 @@
 #include "scrollareawidgetcontainer_p.h"
 #include "stackedwidgetwidgetcontainer_p.h"
 #include "toolbarwidgetcontainer_p.h"
+#include "declarativepixmap_p.h"
```

Comment:
I think the only change we need right now is fixing order of includes (should be alphabetically sorted)